src/app/pages/register/register.component.ts

```diff
@@ -0,0 +1,206 @@
+import { Component, inject } from '@angular/core';
+import { Router, ActivatedRoute, RouterLink } from '@angular/router';
+import {
+  FormBuilder,
+  FormGroup,
+  Validators,
+  ReactiveFormsModule,
+  ValidationErrors,
+  FormControl,
+} from '@angular/forms';
+import { CommonModule } from '@angular/common';
+
+interface BundleService {
+  id: string;
+  name: string;
+  termsUrl: string;
+}
+
+interface Bundle {
+  id: string;
+  name: string;
+  services: BundleService[];
+}
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.component.html',
+  imports: [RouterLink, ReactiveFormsModule, CommonModule],
+  styleUrl: './register.component.css',
+})
+export class RegisterComponent {
+  private router = inject(Router);
+  private route = inject(ActivatedRoute);
+  private formBuilder = inject(FormBuilder);
+
+  currentStep = 1;
+  totalSteps = 5;
+
+  bundles: Bundle[] = [
+    {
+      id: 'data-portal-galaxy',
+      name: 'Data Portal and Galaxy',
+      services: [
+        {
```

Comment:
Extract the BPA/Galaxy BundleServices out to separate variables so we don't define them multiple times